app/controllers/buyers/accounts_controller.rb

```diff
@@ -52,6 +52,7 @@ def create
       redirect_to admin_buyers_account_path(@buyer)
     else
       @user = signup_result.user
+      flash.now[:error] = signup_result.errors.messages.without(:account, :user).values.join(', ')
```

Comment:
It would be nice to have a test for the case when there's more than one message joint.